roofit/roofitcore/test/test_lib.h

```diff
@@ -61,7 +62,7 @@ generate_ND_gaussian_pdf_nll(RooWorkspace &w, unsigned int n, unsigned long N_ev
   RooArgSet obs_set;
 
   // create gaussian parameters
-  double mean[n], sigma[n];
+  std::vector<double> mean(n), sigma(n);
```

Comment:
Not a big deal, but all these `std::vector` that you introduced can also be `std::array` here.